dev/bin/generate_docs

```diff
@@ -34,20 +53,48 @@ class GenerateDocs
           sibling_groups[node_depth][parent_id] << mapped_category
         end
       end
-      fail if sibling_groups.empty?
-
-      FileUtils.mkdir_p("#{Application.root}/docs/_data")
+      if sibling_groups.empty?
+        puts "  ⨯ No categories found. Aborting."
+        fail
+      end
 
-      puts "Writing sibling_groups.yml..."
+      puts "  → Writing docs/_data/sibling_groups.yml"
       File.open("#{Application.root}/docs/_data/sibling_groups.yml", "w") do |file|
         file.write(sibling_groups.to_yaml(line_width: 1000))
       end
+      puts "  ✓"
+      puts
+    end
 
-      puts "Writing attributes.yml..."
-      attribute_data = JSON.load_file("#{Application.root}/dist/attributes.json")
+    def generate_search_index(verticals)
+      puts "Preparing search index"
+      puts "  → Parsing categories into search index"
+      search_index = verticals.flat_map do |vertical|
+        vertical['categories'].map do |category|
+          {
+            "title" => category.fetch("name"),
+            "url" => "?categoryId=#{CGI.escapeURIComponent(category.fetch("id"))}",
+            "body" => category.fetch("full_name"),
```

Comment:
Nit: the doc says `body` is optional. We could use `full_name` as the `title` and remove `body`.

Reply:
Yeah I was torn on if search results with full name ended up being _too much_, but probably necessary given result quality